spk/htop/Makefile

```diff
@@ -0,0 +1,23 @@
+SPK_NAME = htop
+SPK_VERS = 1.0.3
+SPK_REV = 1
+SPK_ICON = src/htop.png
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = An interactive process viewer for Linux (the package is intended for DSM-4.3 or earlier where it's not available)
```

Comment:
This is just nitpicking, but you can change the `(the package[..] available)` to a separate sentence: 
`An interactive process viewer for Linux. The package is intended for DSM-4.3 or earlier where it's not available.`

Btw, I believe that should be DSM 5.0 there, not 4.3, right? If I'm not mistaken, DSM5.1 was the first version with `htop` (but go ahead and verify that if you can)